### PR TITLE
SER-795 Added product_person field

### PR DESCRIPTION
**Pull request by jcellary**
https://wikia-inc.atlassian.net/browse/SER-795

Added product person field to all crowdin config files

@pwojcik86

### CODE REVIEW

File: extensions/wikia/AchievementsII/crowdin.conf [hunk 1]
```diff
@@ -1,4 +1,5 @@
 [crowdin.project]
 projectid = "AchievementsII"
+product_person = "mike.holmes"
```
Comment (pwojcik86) [inline extensions/wikia/AchievementsII/crowdin.conf:3]: I can't find this user in jira

**Review by pwojcik86 — APPROVED** [review]
Looks ok, but I couldn't find Mycroft in jira for some reason

File: extensions/wikia/AchievementsII/crowdin.conf (hunk 1 above)
Comment (sqreek) [reply extensions/wikia/AchievementsII/crowdin.conf:3]: it's userId and username for Mycroft Holmes

Comment (jcellary) [reply extensions/wikia/AchievementsII/crowdin.conf:3]: yup, I looked up all user names: https://wikia-inc.atlassian.net/secure/ViewProfile.jspa?name=mike.holmes

**Pull request merged**